### DISCUSSION TITLE
More realistic FollowupProvider contextual examples?

### DISCUSSION THREAD

**Discussion by InTheCloudDan** [microsoft/vscode-discussions#956] [Extension Development QnA] [answered]
The `Meow` example - https://github.com/microsoft/vscode-extension-samples/blob/main/chat-agent-sample/src/extension.ts#L90-L96 is helpful for the getting started from 0, but when looking at building a contextual based suggestion for the next question it would help to have a little more guidance/better examples on how this could be achieved.

**Comment by isidorn** [comment, marked as answer]
I agree! I would like to improve this and plan to work on it this week.
@intheclouddan let me know if you have more feedback. Also PRs are welcome for the chat sample! Thank you
